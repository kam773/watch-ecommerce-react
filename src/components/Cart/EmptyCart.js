import React from 'react'

export default function EmptyCart() {
  return (
      <div className="container my-5">
          <div className="row my-5">
              <div className="col-10 mx-auto text-center text-title my-5">
                  <h1>Your cart is currently empty</h1>
              </div>
          </div>
      </div>
  )
}
