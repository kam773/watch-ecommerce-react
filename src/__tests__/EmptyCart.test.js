import React from 'react';
import EmptyCart from '../components/Cart/EmptyCart';
import {shallow, mount} from 'enzyme';
import renderer from 'react-test-renderer';

describe("CartColumnst", () => {
    it('render without crashes', () => {
        const tree = renderer.create(<EmptyCart />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('should render h1 tag', () => {
        const props = {
            text: 'Your cart is currently empty'
        }
        const wrapper = mount(<EmptyCart {...props}/>)
        expect(wrapper.text()).toEqual("Your cart is currently empty");
    })
    it('has a div with class', () => {
        const wrapper = shallow(<EmptyCart />)
        expect(wrapper.find('div').exists()).toEqual(true);
    });
    it('has a heading tag', () => {
        const wrapper = shallow(<EmptyCart />)
        expect(wrapper.find('h1').exists()).toEqual(true);
    });
});
