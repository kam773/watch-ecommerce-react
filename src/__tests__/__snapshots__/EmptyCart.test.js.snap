// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartColumnst render without crashes 1`] = `
<div
  className="container my-5"
>
  <div
    className="row my-5"
  >
    <div
      className="col-10 mx-auto text-center text-title my-5"
    >
      <h1>
        Your cart is currently empty
      </h1>
    </div>
  </div>
</div>
`;
